### DISCUSSION TITLE
regards

### DISCUSSION THREAD

**Discussion by iambumblehead** [nevalang/neva#796] [Other]
I'm just a lurker who has been following for a few months and want to say that nevalang is looking incredible. I browsed recent diffs today including the tutorial and examples and am greatly inspired! My appreciation and regards to you,

Chris

**Comment by emil14** [comment]
@iambumblehead thank you for these words, they give me strength to continue!

I wish Nevalang could be ready for at least small pet-projects, but it's still a prototype. However, we have a clear vision, what to do...

Join our [community](https://github.com/nevalang/neva?tab=readme-ov-file#-community) if you want, we need people like you.

**Reply by iambumblehead** [reply]
Thank you. I will join one of the groups, but may not be useful due to lack of free time.

**Reply by emil14** [reply]
Sure
